index.html

```diff
@@ -4999,9 +4999,9 @@ <h3>HTML Attribute State and Property Mappings</h3>
             <tr tabindex="-1" id="att-label">
               <th>`label`</th>
               <td class="elements">
-                <a data-cite="html/#attr-optgroup-label">`optgroup`</a>;
-                <a data-cite="html/#attr-option-label">`option`</a>;
-                <a data-cite="html/#attr-track-label">`track`</a>
+              	<a data-cite="html/form-elements.html#attr-optgroup-label">`optgroup`</a>;
```

Comment:
This is safer and more clear:
 
```suggestion
              	[^optgroup^]'s [^optgroup/label^];
```

Reply:
this would create links within buttons per my other comment, and would create a lot of redundancy to the attribute's name.

e.g., label [ optgroup's label; option's label; track's label ] 

that would be unnecessary

Reply:
Sorry, you've misunderstood what I'm trying to say/do... it's bad practice to use `data-cite=` in the manner you are doing above (i.e., hard links into HTML). That makes the links fragile and they will eventually break. 

So, I guess what you need is a way of aliasing `[^optgroup/label|optgroup^]`.

Reply:
I'll try to land this soon - https://github.com/w3c/respec/issues/3654 . Then all these can hopefully be changed.